src/core/qgsziputils.cpp

```diff
@@ -85,6 +85,14 @@ bool QgsZipUtils::unzip( const QString &zipFilename, const QString &dir, QString
         if ( zip_fread( file, buf, len ) != -1 )
         {
           QFileInfo newFile( QDir( dir ), QString( stat.name ) );
+          QString fileName( stat.name );
```

Comment:
Would be good to move one line up so that the `fileName` variable can be reused when creating QFileInfo (and we do not need to wrap the file name into QString twice